.gitattributes

```diff
@@ -0,0 +1,14 @@
+# TODO
+# Execute the following command in your local project
+# directory to enable automatic merge resolution:
```

Comment:
I think we should add here some more explanations. How about that.
... to register the shell command "true" as merge driver "ours" that does just nothing during a git merge.

Reply:
Ping! 
This command does not enable automatic merge resolution.
It sets a shell script that is used for all merge=ours files.
In this case the script "true" dies not touch anything, just returns successful.